packages/desktop/components/modals/AccountActionsMenu.svelte

```diff
@@ -24,6 +28,29 @@
         modal?.close()
     }
 
+    function onVerifyAddressClick(): void {
+        const ADDRESS_INDEX = 0
+        checkOrConnectLedger(() => {
+            try {
+                if ($profileManager && $selectedAccount && $isActiveLedgerProfile) {
+                    $profileManager.generateEd25519Address($selectedAccount.index, ADDRESS_INDEX, {
+                        internal: false,
+                        ledgerNanoPrompt: true,
+                    })
+                    showAppNotification({
+                        type: 'info',
+                        message: localize('views.generateNewLedgerAddress.confirmedBody'),
```

Comment:
instead of this message, i would hint the user to please check the ledger device and verify that the deposit address matches the one displayed on the ledger device